app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except VaccineError:
+            return "All friends should be vaccinated"
+
+    if masks_to_buy > 0:
```

Comment:
Code Efficiency: There is no need to compare `masks_to_buy` to `0`. You can just check if `masks_to_buy` is truthy. This is because in Python, `0` is falsy and any other number is truthy. So, you can simplify your code like this: `if masks_to_buy:`